source/frontend/src/features/projects/list/ProjectFilter/ProjectFilter.tsx

```diff
@@ -75,10 +75,10 @@ export const ProjectFilter: React.FunctionComponent<
 
             <Col xl="4">
               <Row>
-                <Col xl="4">
```

Comment:
This forces white space at smaller resolutions when it should be used for fields.